examples/python/cloud-function-write-to-bigquery/main.py

```diff
@@ -0,0 +1,55 @@
+from google.cloud import bigquery
+import looker_sdk
+client = bigquery.Client()
+sdk = looker_sdk.init40()
+
+def main(request):
+  get_data_from_looker()
+  write_to_file()
+  load_to_bq()
+  return("Successfully loaded data from Looker to BigQuery")
+
+def get_data_from_looker(query_id=1):
+  query = sdk.run_query(
+    query_id=query_id,
+    result_format="csv",
+    limit= 5000
+  )  
+  print("Successfully retrieved data from Looker")
+  return query
+
+def write_to_file():
+  data = get_data_from_looker() 
+  ### Transform headers and data ("User Name" to become "User_Name")
+  cnt = 0
+  for i in data: 
+    if i == "\n":
+        break
+    else: 
+        cnt += 1
+  header = data[:cnt]
+  header_to_write = header.replace(" ", "_")
+  data_to_write = data[cnt:]
```

Comment:
is this because bq column names can't have spaces? If so, might mention that in the comment.

Reply:
Thank you! I added a comment inside the function and at the top of the script